tests/TestCase.php

```diff
@@ -9,13 +9,13 @@ abstract class TestCase extends OrchestraTestCase
     /**
      * Setup the test environment.
      */
-    public function setUp()
+    public function setUp() : void
     {
         parent::setUp();
         // migrations only for testing purpose
         $this->loadMigrationsFrom([
             '--database' => 'testbench',
-            '--realpath' => __DIR__.'/migrations',
+            '--path' => __DIR__.'/migrations',
```

Comment:
Is realpath removed from 6.0? However, reading the docs from testbench, it seems `$this->loadMigrationsFrom(__DIR__ . '/migrations');` seems to be enough, since this is only running in tests and the env should already be set to the correct database.. I'm not quite sure of this actually.. Could you test this? Or else I can have a look at it tomorrow

Reply:
In orchestra/testbench 4.0 the --realpath option was changed to only accept boolean values: https://github.com/orchestral/testbench-core/blob/4.x/src/Concerns/WithLoadMigrationsFrom.php#L21-L23

Without the --path option set, I'm getting "no such table" errors when testing.